nats/subsz.yml

```diff
@@ -0,0 +1,60 @@
+type: json
+data:
+  url: /subsz
+  server: ${NATS_HOSTPORT}
+
+observation:
+  - name: subsz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: subscriptions
+        source: num_subscriptions
+        type: long_gauge
+        label: subscriptions
+        description: The number of subscriptions
+
+      - name: inserts
+        source: num_inserts
+        type: long_gauge
+        label: inserts
+        description: The number of insert operations in subscriptions list
+
+      - name: removes
+        source: num_removes
+        type: long_gauge
+        label: removes
+        description: The number of remove operations in subscriptions list
+
+      - name: matches
+        source: num_matches
+        type: long_gauge
+        label: matches
+        description: The number of times a match is found for a subscription
+
+      - name: cache
+        source: num_cache
+        type: long_gauge
+        label: cache
+        description: The number of result sets in the cache
+
+      - name: cache.hit.rate
+        source: cache_hit_rate
+        type: double_gauge
+        label: cache hit rate
+        description: The rate matches are being retrieved from cache
```

Comment:
what is the unit for this rate metrics - `%` or `hits/sec`?  Can we get the actual hit count & num requests as a metric?

Reply:
it's value range is [0-1], so it is %, in this case should i multiple value by 100, to change unit to '%'

Reply:
OK. I also see that the actual cache hit metric is not exposed (to calculate the rate ourselves). Please mention that the value range is from 0-1 in the description.

Reply:
OK. Done.

Reply:
i add value range (0-1) to metric description.